config_src/mct_driver/mom_surface_forcing_mct.F90

```diff
@@ -313,6 +315,9 @@ subroutine convert_IOB_to_fluxes(IOB, fluxes, index_bounds, Time, valid_time, G,
 
     if (restore_temp) call safe_alloc_ptr(fluxes%heat_added,isd,ied,jsd,jed)
 
+    call safe_alloc_ptr(fluxes%ice_fraction,isd,ied,jsd,jed)
+    call safe_alloc_ptr(fluxes%u10_sqr,isd,ied,jsd,jed)
```

Comment:
Is there a way to detect whether these fluxes are intended to be set by the coupler? I can envision someone wanting to use these fields, but they get filled with zeros.

Reply:
I believe the CESM coupler will always pass these fields to POP. Once we move towards supporting the GFDL driver (which won't be using code in `config_src/mct_driver`), we'll want to figure out what to do when these fields aren't available. (I suspect "abort if MARBL is enabled but the fields aren't coming in.")

Reply:
Presumably, this will be implemented in the NUOPC cap as well at some point. Within that framework, I believe it is possible to "detect" if a certain flux is set by the coupler.